apps/frontend/src/app/dashboard/dashboard/dashboard.component.html

```diff
@@ -58,5 +58,55 @@ <h1 class="mb-4">
         </li>
       </ul>
     </div>
+    <div class="col-md-3">
+      @if (!loggedIn) {
+        <div class="card">
+          <div class="card-body">
+            <img class="card-img" src="/assets/artwork/account.svg" alt="Account" width="100%">
+            <h5 class="card-title">
+              Sign In to Apollusia
```

Comment:
We have inconsistencies with the usage of Sign in / Log in.
It would be advisable that we maybe follow some guides listed in [this discussion](https://ux.stackexchange.com/questions/1080/using-sign-in-vs-using-log-in).

Some interesting perspectives on that topic:
> In "[Homepage Usability](http://www.useit.com/homepageusability/)", Jakob Nielsen (together with Marie Tahir, 2002, p. 53) recommends the use of "sign in" / "sign out" over "log in" / "log out". This is empirically based on a survey of several large-scale websites and thus supports OP's "more common" argument.

Another one:
> I think that this [article about the “Sign Up” button](http://www.gkogan.co/blog/stop-asking-me-to-sign-up/) is interesting, the author changed the “Sign Up” button to “Try it Free” and clicks increased by 212%. His thesis is that the standard “Sign Up” buttons don’t work because “they ask for blind commitment” and “do not offer any value”.

Companies like Google seem to prefer Sign in. So i suggest we use it too, since they probably did their research.

Reply:
What should we change? Replacing this Log In with Sign In looks weird because of the repeated word

<img width="294" alt="Screenshot 2024-07-27 at 16 09 31" src="https://github.com/user-attachments/assets/e19584cd-dcb3-4406-837c-5a07f3bc8458">